public/components/incontext_insight/index.scss

```diff
@@ -109,17 +109,30 @@
   border: none;
 }
 
-.incontextInsightGeneratePopoverFooter{
-  padding-block: 0 !important;
+.incontextInsightGeneratePopoverFooter {
+  padding: 0 $euiSizeS !important;
   margin: 0 !important;
   border: none;
+  display: flex;
+  align-items: center;
+
+  .buttonGroupDivider {
+    height: 24px;
+    border-left: 1px solid rgba(0, 0, 0, 0.2);
```

Comment:
I guess we have variables($ouiBorderThing) for borders. Explicit color here may not work in dark mode.

Reply:
Done